ui/src/Components/Search/Search.js

```diff
@@ -0,0 +1,47 @@
+import { useState, useCallback } from 'react';
+import { FEATURE_FLAGS } from '../../Mock/featureFlags';
+import { useFilterContext } from '../../Context/filter_context';
```

Comment:
We should keep only `useFilter` and remove `context`. useFilter is enough

Reply:
renamed as per suggestion